event_project/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module allows you to create retro planning for managing your events.
```

Comment:
I wish we had a better description.
Shouldn't "retro planning" be a generic Project feature reused for this use case?